platform-sdk/platform-apps/tests/MigrationTestingTool/src/main/java/com/swirlds/demo/migration/MigrationTestingToolState.java

```diff
@@ -198,14 +203,29 @@ protected void setVirtualMap(final VirtualMap<AccountVirtualMapKey, AccountVirtu
      * Do genesis initialization.
      */
     private void genesisInit(final Platform platform) {
+        final Configuration configuration = ConfigurationBuilder.create()
```

Comment:
this just uses default configuration, not config values loaded from files

Reply:
Please see: https://github.com/hashgraph/hedera-services/pull/15562/commits/913fc5060af66d0e6ca93d23201e109ec09f47e2